jupyter_server_proxy/handlers.py

```diff
@@ -444,7 +446,8 @@ def get_timeout(self):
         return 5
 
     async def _http_ready_func(self, p):
-        url = 'http://localhost:{}'.format(self.port)
+        # Use 127.0.0.1 instead of localhost, since it can fail on windows
+        url = 'http://127.0.0.1:{}'.format(self.port)
```

Comment:
Could a few others review this change? I've run into problems with `localhost` vs `127.0.0.1` before on Linux, so we should be careful not to inadvertently break someone else. For example how does this behave with IPv6? Should we actually make this configurable or is that adding unnecessary complexity?